docs/advisors/checks/mongodb-connection-spike.md

```diff
@@ -0,0 +1,49 @@
+# MongoDB Connections sudden spike
+
+## Description
+This check returns a warning if there is an increase in the number of connections that is higher than 50% of the most recent or normal number of connections.
```

Comment:
```suggestion
This check warns if the number of connections increases by 50% comparted to the most recent or normal number of connections.
```

Reply:
@catalinaadam 
1. There seems to be a typo. Instead of  "comparted" it should be "compared".
2. It statement should be more than 50% like this --- "increases by more than 50% compared to".